### PR TITLE
set up new structure

### PR DESCRIPTION
**Pull request by nardonykolyszyn**

### CODE REVIEW

File: lib/mercadopago/api/cards.rb [hunk 1]
```diff
@@ -3,31 +3,33 @@
...
+      # It retrieves all cards from a customer
+      def retrieve_customer_cards(customer_id)
+        response = connection.get customer_cards_endpoint(customer_id) do |req|
+          req.params['access_token'] = @access_token
+        end
+        response = process_response(response)
+        OpenStruct.new(success?: true, body: response)
+      rescue Faraday::ClientError => exception
+        OpenStruct.new(success?: false, message: 'Los datos no son correctos', details: exception.response[:body].to_s)
+      rescue Faraday::Error::TimeoutError, Faraday::ConnectionFailed, Timeout::Error => e
```
Comment (nardonykolyszyn) [inline lib/mercadopago/api/cards.rb:24]: Useless assignment to variable - `e`.

File: lib/mercadopago/api/cards.rb [hunk 2]
```diff
@@ -3,31 +3,33 @@
...
+        'https://api.mercadopago.com'
+      end
 
-    # GET /v1/customers/:customer_id/cards
-    # It retrieves all cards from a customer
-    def retrieve_customer_cards(customer_id)
-      response = connection.get customer_cards_endpoint(customer_id) do |req|
-        req.params['access_token'] = @access_token
+      # GET /v1/customers/:customer_id/cards
+      # It retrieves all cards from a customer
```
Comment (nardonykolyszyn) [inline lib/mercadopago/api/cards.rb]: Missing space after `#`.

File: lib/mercadopago/base.rb [hunk 3]
```diff
@@ -2,10 +2,10 @@
 
 require 'faraday/request_id'
 require 'faraday_middleware'
-require './active_support/deep_symbolize_keys'
+require_relative 'deep_symbolize_keys'
 
 module MercadoPago
-  
+
```
Comment (nardonykolyszyn) [inline lib/mercadopago/base.rb]: Extra empty line detected at module body beginning.

File: lib/mercadopago.rb [hunk 4]
```diff
@@ -1,6 +1,4 @@
 # frozen_string_literal: true
 
-module MercadoPago
-  require 'mercadopago/base'
-  require 'mercadopago/api'
-end
\ No newline at end of file
+require 'mercadopago/base'
```
Comment (nardonykolyszyn) [inline lib/mercadopago.rb:3]: Final newline missing.

File: lib/mercadopago/api/clients.rb [hunk 5]
```diff
@@ -3,45 +3,46 @@
...
 module MercadoPago
-  class Clients < Base
-
-    # Override parent class method
-    # Place here main base URL
-    def service_url
-      'https://api.mercadopago.com'
-    end
+  module API
+    class Clients < Base
```
Comment (nardonykolyszyn) [inline lib/mercadopago/api/clients.rb:7]: MercadoPago::API::Clients assumes too much for instance variable '@access_token'

File: lib/mercadopago/api/cards.rb (hunk 1 above)
Comment (nardonykolyszyn) [inline lib/mercadopago/api/cards.rb:24]: MercadoPago::API::Cards#retrieve_customer_cards has the variable name 'e'

[Read more about it here.](https://github.com/troessner/reek/blob/master/docs/Uncommunicative-Variable-Name.md)

File: lib/mercadopago/api/cards.rb [hunk 6]
```diff
@@ -3,31 +3,33 @@
...
 
 module MercadoPago
-  class Cards < Base
-    # Override parent class method
-    # Place here main base URL
-    def service_url
-      'https://api.mercadopago.com'
-    end
+  module API
+    class Cards < Base
```
Comment (nardonykolyszyn) [inline lib/mercadopago/api/cards.rb:7]: MercadoPago::API::Cards assumes too much for instance variable '@access_token'

**Comment by devpolish** [comment]
Ebert has finished reviewing this Pull Request and has found:

* 9 possible new issues (including those that may have been commented here).
* 8 fixed issues! 🎉

You can see more details about this review at https://ebertapp.io/github/devpolish/mercadopago-ruby/pulls/1.

**Comment by devpolish** [comment]
Ebert has finished reviewing this Pull Request and has found:

* 7 possible new issues (including those that may have been commented here).
* 12 fixed issues! 🎉

You can see more details about this review at https://ebertapp.io/github/devpolish/mercadopago-ruby/pulls/1.

File: lib/mercadopago/api.rb [hunk 7]
```diff
@@ -1,11 +1,11 @@
...
-require 'api/clients'
+require 'mercadopago/api/cards'
+require 'mercadopago/api/clients'
 
 module MercadoPago
   module API
-    include MercadoPago::Cards
-    include MercadoPago::Clients
+    class Cards; end;
+    class Clients; end;
```
Comment (nardonykolyszyn) [inline lib/mercadopago/api.rb:9]: Do not use semicolons to terminate expressions.

File: lib/mercadopago/api.rb [hunk 8]
```diff
@@ -1,11 +1,11 @@
...
-require 'api/cards'
-require 'api/clients'
+require 'mercadopago/api/cards'
+require 'mercadopago/api/clients'
 
 module MercadoPago
   module API
-    include MercadoPago::Cards
-    include MercadoPago::Clients
+    class Cards; end;
```
Comment (nardonykolyszyn) [inline lib/mercadopago/api.rb:8]: Do not use semicolons to terminate expressions.

**Comment by nardonykolyszyn** [comment]
Ebert has finished reviewing this Pull Request and has found:

* 9 possible new issues (including those that may have been commented here).
* 12 fixed issues! 🎉

But beware that this branch is **[1 commit behind the `devpolish:master` branch](https://github.com/devpolish/mercadopago-ruby/compare/devpolish:project_structure...devpolish:master)**, and a review of an up to date branch would produce more accurate results.

You can see more details about this review at https://ebertapp.io/github/devpolish/mercadopago-ruby/pulls/1.

**Pull request merged**